src/supertux/menu/joystick_menu.cpp

```diff
@@ -187,31 +187,31 @@ JoystickMenu::refresh_menu_item(Controller::Control id)
     switch (hat_dir)
     {
       case SDL_HAT_UP:
-        name = "Hat Up";
+        name = _("Hat Up");
         break;
 
       case SDL_HAT_DOWN:
-        name = "Hat Down";
+        name = _("Hat Down");
         break;
 
       case SDL_HAT_LEFT:
-        name = "Hat Left";
+        name = _("Hat Left");
         break;
 
       case SDL_HAT_RIGHT:
-        name = "Hat Right";
+        name = _("Hat Right");
         break;
 
       default:
-        name = "Unknown hat_dir";
+        name = _("Unknown hat_dir");
```

Comment:
I wonder whether this is a user-visible string, after all it has _hat_dir_, which appears to be a variable. Seems weird. Can anyone shed some light onto this?